pages/demo/html/linkuri-care-contin-bucati-mari-de-continut.tsx

```diff
@@ -7,18 +7,16 @@ import Header from '~/components/Header';
 import Footer from '~/components/Footer';
```

Comment:
Same as in the previous files.

We should only modify where to replace with `<FormattedText>`.